lms/djangoapps/course_api/blocks/toggles.py

```diff
@@ -8,19 +8,18 @@
 
 COURSE_BLOCKS_API_NAMESPACE = WaffleFlagNamespace(name=u'course_blocks_api')
 
-"""
-.. toggle_name: course_blocks_api.hide_access_denials
-.. toggle_implementation: WaffleFlag
-.. toggle_default: False
-.. toggle_description: Waffle flag to hide access denial messages in the course blocks.
-.. toggle_category: course api
-.. toggle_use_cases: incremental_release, open_edx
-.. toggle_creation_date: ??
-.. toggle_expiration_date: None
-.. toggle_warnings: None
-.. toggle_tickets: ??
-.. toggle_status: supported
-"""
+# .. toggle_name: course_blocks_api.hide_access_denials
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to hide access denial messages in the course
```

Comment:
@regisb: I wouldn't wrap unless the linter is complaining or you want something like multiple paragraphs.